lib/eyaml/encryption_manager.rb

```diff
@@ -0,0 +1,96 @@
+class EYAML
+  class EncryptionManager
+    FORMAT_REGEX = /\AEJ\[(?<version>[^:]+):(?<session_public_key>[^:]+):(?<nonce>[^:]+):(?<text>[^\]]+)\]\z/
+    FORMAT_VERSION = "1"
+
+    class << self
+      def generate_key
```

Comment:
you call this new_keypair above but generate_key here. Maybe write a test 😄

Reply:
😅